unison-cli/src/Unison/Codebase/Editor/HandleInput/AuthLogin.hs

```diff
@@ -1,17 +1,22 @@
-{-# LANGUAGE RecordWildCards #-}
-
 module Unison.Codebase.Editor.HandleInput.AuthLogin (authLogin) where
 
 import Control.Monad.Reader
+import qualified Data.Text as Text
+import System.IO.Unsafe (unsafePerformIO)
 import Unison.Auth.OAuth
 import Unison.Auth.Types (Host (..))
 import Unison.Codebase.Editor.HandleInput.LoopState
 import Unison.Codebase.Editor.Output (Output (CredentialFailureMsg, Success))
 import Unison.Prelude
 import qualified UnliftIO
+import UnliftIO.Environment (lookupEnv)
 
 defaultShareHost :: Host
-defaultShareHost = Host "enlil.unison-lang.org"
+defaultShareHost = unsafePerformIO $ do
+  lookupEnv "UNISON_SHARE_HOST" <&> \case
+    Nothing -> Host "share.unison-lang.org"
```

Comment:
I think this is where we plan to put it, but we can fix it once we deploy prod, but before we release the new ucm milestone